dev/inc/SharedHelpers.h

```diff
@@ -193,6 +193,15 @@ class SharedHelpers
         }
     }
 
+    static bool EvaluateNullableBool(winrt::IReference<bool> const& nullableBool)
+    {
```

Comment:
EvaluateNullableBool is a non-intuitive name to me. Evaluate it to what?

Maybe "NullableBoolIsTrue" ? Since it's checking to see if the IReference is non-null and contains the value true?

Reply:
What about just IsTrue since the parameter is a NullableBool